### ISSUE TITLE
Delete all tags via GUI

### ISSUE THREAD

**Issue by bartblaze** [MISP/MISP#7331]
**Is your feature request related to a problem? Please describe.**
Sometimes, too much tagging is... too much. Would be nice if we have a possibility via the GUI to remove all tags for an event.

**Describe the solution you'd like**
A simple "DEL" button could do the trick - with additional messagebox for confirmation.
![image](https://user-images.githubusercontent.com/3075118/114527027-cf550a00-9c47-11eb-968d-8d15cbbbcd8d.png)

**Describe alternatives you've considered**
None.

**Additional context**
Optionally, the messagebox could pop up with 4 options:
[v] Delete all tags
[v] Delete global tags
[v] Delete local tags
[v] Cancel

**Comment by chrisinmtown** [comment]
So this is "remove all tags ON AN EVENT" not nuke all tags defined in the system?

**Comment by bartblaze** [comment]
Yes, correct - the option to remove all tags on an event that you're browsing.
